src/app/components/room/room.service.js

```diff
@@ -15,6 +15,11 @@
       'FeedConnection', 'DataChannelService', 'ActionService', 'jhConfig',
       'ScreenShareService'];
 
+  /**
+   * Service containing all rooms
```

Comment:
If it is service to manage communication with janus, then why it is called Room Service?